docker-compose-ss.yml

```diff
@@ -0,0 +1,99 @@
+x-superset-image: &superset-image apache/superset:${TAG:-latest-dev}
+x-superset-depends-on: &superset-depends-on
+  - db
+  - redis
+x-superset-volumes: &superset-volumes
+  # /app/pythonpath_docker will be appended to the PYTHONPATH in the final container
+  - ./docker/superset:/app/docker
+  - ${SUPERSET_HOME_DATA:-./.data/superset_home}:/app/superset_home
+
+x-environment: &environment
+  - DATABASE_HOST=postgres
+  - DATABASE_DB=superset
+  - DATABASE_PASSWORD=${POSTGRES_PASSWORD:-postgres}
+  - DATABASE_USER=postgres
+  - DATABASE_PORT=5432
+  - DATABASE_DIALECT=postgresql
+
+  - POSTGRES_DB=superset
+  - POSTGRES_USER=postgres
+  - POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-postgres}
+  - PYTHONPATH=/app/pythonpath:/app/docker/pythonpath_dev
+  - REDIS_HOST=redis
+  - REDIS_PORT=6379
+  - FLASK_ENV=production
+  - SUPERSET_ENV=production
+  - SUPERSET_LOAD_EXAMPLES=no
+  - SUPERSET_SECRET_KEY=${SUPERSET_SECRET_KEY:-secretkey}
+  - CYPRESS_CONFIG=false
+  - SUPERSET_PORT=8088
+  - MAPBOX_API_KEY=${MAPBOX_API_KEY:-}
+
+version: "3.9"
+services:
+  redis:
```

Comment:
oh superset depends on and leverage redis? The componet setup is getting complex.